xbmc/pvr/guilib/PVRGUIActionsPlayback.cpp

```diff
@@ -214,6 +214,7 @@ bool CPVRGUIActionsPlayback::PlayEpgTag(
     return false;
 
   CPVRStreamProperties props;
+  client->StreamClosed();
```

Comment:
add error handling

Reply:
Added a log statement in the event the clint returns an error. Is that sufficient?